CHANGELOG.rst

```diff
@@ -3,6 +3,14 @@ Changelog
 =========
 
 
+[202210xx]
+==========
```

Comment:
Generally we use unreleased, but it's okay if you wanna leave it this way. My guess is 202211xx for the actual next release :wink:

Reply:
Oh, I thought these were change log entry dates, not release tags. I can fix that. I just remove the date entry, right?

Reply:
Yah and replace with "Unreleased"

Reply:
addressed